src/coreclr/hosts/corerun/corerun.hpp

```diff
@@ -471,6 +471,7 @@ namespace pal
         {
             // We have data. At this point we can likely make a strong decision.
             const char tracer_pid_name[] = "TracerPid:";
+            // CodeQL [SM01932] status is of fixed size and [sizeof(status) -1] is specified for read .
```

Comment:
I wonder if we could instead make it happy by adding `status[bytes_read] = 0;`. We could then remove the zero initialization of the status array, which could also make performance a tiny bit better.

Reply:
We actually do exactly the same thing in pal.cpp:
https://github.com/dotnet/runtime/blob/66e45ef152002a2e1cbea939ded6643dfa48307c/src/coreclr/pal/src/init/pal.cpp#L788-L814

Reply:
ok interesting, will make that change then.